### PR TITLE
Versioned artifacts

### PR DESCRIPTION
**Pull request by mohitsoni**
Adds additional scripts to upload versioned bootstrap.zip and executor.zip as part of the release process. These artifacts are meant to be consumed by frameworks, especially, the ones that reside outside the monorepo.

### CODE REVIEW

**Comment by mohitsoni** [comment]
retest this please

File: release.sh [hunk 1]
```diff
@@ -10,5 +10,9 @@ cd $REPO_ROOT_DIR
 ./build.sh
 ./gradlew publish
 
+# Scripts below expects S3_BUCKET and SDK_VERSION env vars. To be supplied by Jenkins.
```
Comment (benclarkwood) [inline release.sh]: These are existing envvars, right?

Comment (benclarkwood) [inline release.sh]: Looks like it, and they have sane defaults 👍

Comment (mohitsoni) [inline release.sh]: Yes. They are passed in from the Build job. And, like you noticed they have sane defaults.

**Comment by nickbp** [comment]
retest this please (looks like host resolv flake)

File: tools/build_publishable.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```
Comment (mgummelt) [inline tools/build_publishable.sh]: Let's get in the habit of documenting what a script does at the top of the script.  The name alone doesn't tell me.

**Pull request merged**